lib/go-tc/servers.go

```diff
@@ -718,6 +757,15 @@ type ServerNullableV2 struct {
 	IP6IsService *bool `json:"ip6IsService" db:"ip6_address_is_service"`
 }
 
+type ServerNullableV40 struct {
```

Comment:
I was updating all structs for V40 and hence created this one, even though I ended up not using it. This was a left over from earlier commits. It will be removed.